src/widgets/universedisplay.cpp

```diff
@@ -99,7 +139,7 @@ void UniverseDisplay::levelsChanged()
         }
         else
         {
-            setCellColor(i, Qt::white);
+            setCellColor(i, Preferences::Instance().GetTheme() == Themes::LIGHT ? Qt::white : Qt::black);
```

Comment:
Ideally this would be a switch statement, with no default, as we _might_ add more themes later.

Reply:
YAGNI.
There's a lot of other places that would need changing to support a third theme, I think it best to do it this way until such a time as that happens.